lib/Test/Nginx/Util.pm

```diff
@@ -918,6 +918,12 @@ http {
     default_type text/plain;
     keepalive_timeout  68;
 
+    client_body_temp_path "$ServRoot/client_body_temp";
+    proxy_temp_path "$ServRoot/proxy_temp";
+    fastcgi_temp_path "$ServRoot/fastcgi_temp";
+    scgi_temp_path "$ServRoot/scgi_temp";
+    uwsgi_temp_path "$ServRoot/uwsgi_temp";
```

Comment:
Shall we put these to the end of the `http {}` block? Your additions here will change the `nginx.conf` line numbers of the user configuration snippets, and thus will break existing tests checking line numbers of `nginx.conf` in some error messages.